central/config/datastore/datastore_test.go

```diff
@@ -50,24 +50,106 @@ func (s *configDataStoreTestSuite) TearDownTest() {
 	s.mockCtrl.Finish()
 }
 
+var (
+	sampleConfig = &storage.Config{
+		PublicConfig: &storage.PublicConfig{
+			LoginNotice: &storage.LoginNotice{
+				Enabled: false,
+				Text: "You step onto the road, and if you don't keep your feet, " +
+					"there's no knowing where you might be swept off to.",
```

Comment:
Yesss, more life in tests!

Reply:
Although I did not read Dr. Seuss as a child, here is my favorite for tests:

> From there to here, from here to there, funny things are everywhere!